package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@headspace/zuora-request-service",
-  "version": "1.1.2-rc0",
+  "version": "1.1.3-rc0",
```

Comment:
We should try to avoid incrementing the minor version if we're going to do release candidate builds like `rc0`, `rc1`, etc